internal/services/loganalytics/client/client.go

```diff
@@ -3,6 +3,7 @@ package client
 import (
 	"github.com/Azure/azure-sdk-for-go/services/operationalinsights/mgmt/2020-08-01/operationalinsights"
 	"github.com/Azure/azure-sdk-for-go/services/preview/operationsmanagement/mgmt/2015-11-01-preview/operationsmanagement"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/operationalinsights/2020-08-01/workspaces"
```

Comment:
could we move everything over to the new SDK instead of just adding another one?

Reply:
@katbyte , Nope. Because each service in OperationalInsights is their separate folder. So they are not related with each other. If you intend to switch all of them to new SDK, suggest to submit another PR for your request.

Reply:
👍  - as there is no existing workspaces SDK this should be ok